### PR TITLE
[COMMITTERS] Update committers list

### PR DESCRIPTION
**Pull request by andreaskurth**
Add new committers approved by TC on 2024-09-05.  Congrats and thanks for your contributions!

### CODE REVIEW

**Review by nbdd0121 — APPROVED** [review]
Thanks!

**Review by nasahlpa — APPROVED** [review]
Thanks :)

File: COMMITTERS [hunk 1]
```diff
@@ -33,3 +33,5 @@ Committer list:
 * Pirmin Vogel (vogelpi)
 * James Wainwright (jwnrt)
 * Alex Williams (a-will)
+* Pascal Nasahl (nasahlpa)
```
Comment (mundaym) [inline COMMITTERS:36]: This list should be kept alphabetical (by surname)

Comment (andreaskurth) [reply COMMITTERS]: Thanks, fixed

**Pull request merged**